### PR TITLE
Make edit to the readme file

### PR DESCRIPTION
**Pull request by dagmawidemissie17**
**Description**
This PR will add an explanation to the readme file

### CODE REVIEW

**Review by 1jc — APPROVED** [review]
1jc   jc damo reviewed looks good

**Comment by MarcyAnna** [comment]
Makes sense! I think!

**Review by MarcyAnna — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 # IT115Group1
+
+This is an official repository for IT115Group1.
```
Comment (AntFranco) [inline README.md:3]: This is my comment

**Review by AntFranco — APPROVED** [review]
I approve- ANtonio

**Pull request merged**